### PR TITLE
BTRX - 333 - Admin manage Dul Modals

### PR DESCRIPTION
**Pull request by LeoForconesi**

### CODE REVIEW

File: src/components/modals/AddDulModal.js [hunk 1]
```diff
@@ -57,87 +107,253 @@ export const AddDulModal = hh(class AddDulModal extends Component {
...
+      return true;
+    } else {
+      this.handleErrors(response);
+    }
+    return false;
+  };
+
+
+  handleErrors(message) {
+    var tle = 'Conflicts to resolve!';
```
Comment (dags) [inline src/components/modals/AddDulModal.js]: what tle means ?

File: src/pages/AdminManageDul.js [hunk 2]
```diff
@@ -19,11 +18,14 @@ class AdminManageDul extends Component {
     this.state = {
       currentPage: 1,
       showModal: false,
+      editMode: false,
```
Comment (dags) [inline src/pages/AdminManageDul.js]: is this still used ?

File: src/pages/AdminManageDul.js [hunk 3]
```diff
@@ -326,11 +335,14 @@ class AdminManageDul extends Component {
                     button({
                       id: election.consentId + "_btn_editDUL",
                       className: "cell-button hover-color",
-                      onClick: this.editDul(election)
+                      onClick: () => this.editDul(election)
```
Comment (dags) [inline src/pages/AdminManageDul.js:337]: review this, every click invokes a new instance ...

File: src/pages/DulReview.js [hunk 4]
```diff
@@ -44,10 +44,11 @@ class DulReview extends Component {
       .then(data => {
         this.setState({election: data});
       });
-    Consent.ConsentResource(this.props.match.params.consentId)
-      .then(data => {
-        this.setState({consent: data});
-      });
+    this.setState({consent: Consent.ConsentResource(this.props.match.params.consentId)});
+      // .then(data => {
```
Comment (dags) [inline src/pages/DulReview.js]: remove not used lines

File: src/components/modals/AddDulModal.js (hunk 1 above)
Comment (LeoForconesi) [reply src/components/modals/AddDulModal.js]: This stands for Title related to the Error alert title. I've change this to a better variable name.

File: src/pages/AdminManageDul.js (hunk 2 above)
Comment (LeoForconesi) [reply src/pages/AdminManageDul.js]: Removed

File: src/pages/AdminManageDul.js (hunk 3 above)
Comment (LeoForconesi) [reply src/pages/AdminManageDul.js:337]: This is done this way due to, onClick listener expects ```this.editDul``` be a function, if we pass the  instance value to the function the app enters into an infinite loop.

**Pull request merged**